### DISCUSSION TITLE
sh1mmer unenroll

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#402] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
SH1MMER is an exploit capable of completely unenrolling enterprise-managed Chromebooks. It was found by the Mercury Workshop team

https://github.com/3kh0/ext-remover/#using-sh1mmer

![image](https://user-images.githubusercontent.com/58097612/212473507-d7b6ae5f-0866-4603-8a48-f549f04c312b.png)

**Comment by Aka-but-nice** [comment]
Sadly, I need a USB, which currently I seem to not have

**Reply by 3kh0** [reply]
buy one, it is not that expensive

**Reply by Aka-but-nice** [reply]
ik

**Comment by rwg100** [comment, marked as answer]
unenrollment is illegal

**Reply by 3kh0** [reply]
*laughs in unenrolled chromebook*

**Reply by rwg100** [reply]
*laughs in i know who you are, where you used to go to school and know whos class you used to be in*

**Reply by rwg100** [reply]
*and you somewhere in georgia because you ditched us and moved somewhere two hours away from atlanta i think idk hahahahahah*

**Reply by 3kh0** [reply]
*laughs in surfshark vpn*

**Reply by rwg100** [reply]
*laughs in why use sharksurf its ass instead use protonvpn*

**Reply by rwg100** [reply]
*also laughs in im not bluffing even though none of the info is relevant since you moved*

**Reply by Aka-but-nice** [reply]
Yo, I'm making a pull request that will change everything I just made a exploit, stay tune...

**Reply by bartholemuew** [reply]
unfortunately we do not care

**Reply by bartholemuew** [reply]
> unfortunately we do not care

just kidding my school i care deeply

**Comment by SomePVZPlayerMatt** [comment]
illegal ahhh

**Comment by Alex2Mayhem** [comment]
> SH1MMER is an exploit capable of completely unenrolling enterprise-managed Chromebooks. It was found by the Mercury Workshop team
> 
> https://github.com/3kh0/ext-remover/#using-sh1mmer
> 
> ![image](https://user-images.githubusercontent.com/58097612/212473507-d7b6ae5f-0866-4603-8a48-f549f04c312b.png)

Clearly states not to unenroll bozo.

**Reply by Aka-but-nice** [reply]
Alr so it's out go to https://github.com/3kh0/ext-remover, and go to New Point Blank, if that doesn't work ty killcurly
